executor/executor.cc

```diff
@@ -567,7 +560,6 @@ int main(int argc, char** argv)
 #endif
 }
 
-#if SYZ_EXECUTOR_USES_SHMEM
 // This method can be invoked as many times as one likes - MMAP_FIXED can overwrite the previous
```

Comment:
I'm not sure this statement is correct after the changes. Guess it doesn't work for asan anymore.

Reply:
I think it still holds. Under asan we don't use MAP_FIXED only during the first mapping (when nothing is mapped yet), on subsequent calls we use MAP_FIXED even under asan and they overwrite previous mappings.